cnf-tests/Dockerfile

```diff
@@ -81,15 +57,16 @@ COPY --from=builder-latency-test-runners /hwlatdetect-runner /usr/bin/hwlatdetec
 COPY --from=builder-latency-test-tools /hwlatdetect /usr/bin/hwlatdetect
 
 COPY --from=oc /go/src/github.com/openshift/oc/oc /usr/bin/oc
-COPY --from=builder /go/src/github.com/openshift-kni/cnf-features-deploy/cnf-tests/bin/cnftests /usr/bin/cnftests
-COPY --from=builder /go/src/github.com/openshift-kni/cnf-features-deploy/cnf-tests/bin/configsuite /usr/bin/configsuite
-COPY --from=builder /go/src/github.com/openshift-kni/cnf-features-deploy/cnf-tests/bin/validationsuite /usr/bin/validationsuite
-COPY --from=builder /go/src/github.com/openshift-kni/cnf-features-deploy/cnf-tests/entrypoint/test-run.sh /usr/bin/test-run.sh
-COPY --from=builder /go/src/github.com/openshift-kni/cnf-features-deploy/cnf-tests/bin/mirror /usr/bin/mirror
-COPY --from=builder /go/src/github.com/openshift-kni/cnf-features-deploy/cnf-tests/mirror/images.json /usr/local/etc/cnf
-COPY --from=builder /go/src/github.com/openshift-kni/cnf-features-deploy/cnf-tests/bin/cnftests-sha.txt /usr/local/etc/cnf
+
+COPY --from=go-builder /app/cnf-tests/bin/mirror /usr/bin/mirror
+COPY --from=go-builder /app/cnf-tests/mirror/images.json /usr/local/etc/cnf
+COPY --from=go-builder /app/cnf-tests/entrypoint/test-run.sh /usr/bin/test-run.sh
+
 COPY tools/oot-driver/ /usr/src/oot-driver/
 
+# Copy test suite
+COPY --from=go-builder /app/cnf-tests/submodules/cluster-node-tuning-operator/build/_output/bin/latency-e2e.test /usr/bin/latency-e2e.test
```

Comment:
why is this needed?

Reply:
This is a binary load file of the latency tools tests. We use it in https://github.com/openshift/cluster-node-tuning-operator/tree/master/test/e2e/performanceprofile/functests/5_latency_testing which AFAIU don't need to export it in cnf-tests

Reply:
Thanks for clarifying. the direction then looks good from my end.